src/redux/user/userSlice.ts

```diff
@@ -1,13 +1,13 @@
-import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
-import axios from 'axios';
+import { createSlice, createAsyncThunk } from "@reduxjs/toolkit";
```

Comment:
The changes in this file were purely formatting changes made by prettier. As long as our prettier config differs we may continue to see these modifications.